docs/cicd/cicd-spans.md

```diff
@@ -0,0 +1,106 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Spans
+--->
+
+# Semantic conventions for CICD spans
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [CICD Spans](#cicd-spans)
+  - [Pipeline run](#pipeline-run)
+  - [Pipeline task run](#pipeline-task-run)
+
+<!-- tocstop -->
+
+## CICD Spans
+
+The conventions described in this section are specific to Continuous Integration / Continuous Deployment (CICD) systems.
+
+**Disclaimer:** These are initial CICD spans and attributes
```

Comment:
Maybe we don't need this disclaimer. I believe this is somewhat implicit by the document status.

Reply:
Removed in 41762ab10dc5eb134c55bd84bedab0dafd8ae9a3